stan/math/prim/mat/fun/cholesky_decompose.hpp

```diff
@@ -31,7 +32,7 @@ Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> cholesky_decompose(
   check_square("cholesky_decompose", "m", m);
   check_symmetric("cholesky_decompose", "m", m);
 #ifdef STAN_OPENCL
-  if (m.rows() > 1250) {
+  if (m.rows() >= opencl_context.tuning_opts("move_to_gpu")) {
```

Comment:
Not sure I understand what "move_to_gpu" means here... is this the size at which we think a matrix is worth moving to the GPU for a cholesky_decomposition? Maybe a slightly more verbose name like `cholesky_size_worth_gpu_transfer` or something like that would be good.

Reply:
> Not sure I understand what "move_to_gpu" means here... is this the size at which we think a matrix is worth moving to the GPU for a cholesky_decomposition?

Yep! I need to ruminate on a better name here. I like yours but would like something a little shorter

Reply:
`cholesky_size_worth_opencl`? Not much shorter tho'